### ISSUE TITLE
ML privatization example text

### ISSUE THREAD

**Issue by Trey-Scheid** [Trey-Scheid/privacy-in-practice#42]
can you turn this sentence (or part of it) into a drop down:

"A data scientist may do this because they don't want to reveal if someone was in a training set."

in the drop down include this e.g.

When ChatGPT 3.5 was released, trained on many datasets publicly and privately available. Clever prompters "Agents" were able to gather SSN's for individuals which the model would produce with perfect accuracy! a differentially private algorithm guarantees that model outputs are not significantly different wether your SSN was in the training data or not.
